### DISCUSSION TITLE
[RISOLTO] err. 401: Unauthorized da Firefox

### DISCUSSION THREAD

**Discussion by wilrino** [indomus/forum#625] [HUB: Home Assistant]
Ciao a tutti, 
mi sta succedendo una cosa strana.
Tanto che ad un certo punto per indagare meglio ho cambiato SD del raspberry, ho _installato da zero Home Assistant OS_ ma il problema non cambia:

Alcune schermate come **file editor, log viewer, terminal** se le apro con il mio pc principale mi danno **errore 401**.
Mentre vanno correttamente se le apro dal telefono (collegato in wifi sulla stessa rete) e addirittura da macchina virtuale, con stesso indirizzo IP!

Premetto che l'installazione sul raspy 4 è fresca, ho solo aggiunto dal supervisor alcuni componenti per vedere se riscontravo questo problema.
Nel file configuration.yaml ho solo aggiunto il sensore per la temperatura della CPU.

C'è qualche altra impostazione che mi sfugge?
Le versioni di firefox che utilizzo sono le stesse (aggiornate all'ultima vers. 91.0.1 64bit)

Nel log ci sono questi errori:
```
2021-08-17 21:59:12 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from PC.local (192.168.2.34). (Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0)
2021-08-17 22:20:56 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from DESKTOP.local (192.168.2.47). (Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0)
2021-08-17 22:42:46 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from CELL.local (192.168.2.185). (Mozilla/5.0 (Android 11; Mobile; rv:91.0) Gecko/91.0 Firefox/91.0)
```
da cui si notano i 3 accessi, ma tra il DESKTOP che è quello che da errore 401 e gli altri non ci sono differenze.

Ho fatto l'accesso a tutti e tre con le stesse credenziali.

![Immagine 2021-08-17 224330](https://user-images.githubusercontent.com/72148374/129801279-0891d574-27fe-4a44-84fb-f7021bf1a2f0.png)

**Comment by wilrino** [comment]
Ulteriore prova:
Aprendo in firefox una nuova finestra anonima (ipotizzando un problema di cookies) il problema 401 si presenta sempre sullo stesso PC.

**Comment by wilrino** [comment]
Ragazzi ho risolto fortunatamente (Ieri ci ho perso un giorno e ho reinstallato HA da zero 🙈

Semplicemente andando in **firefox**, sull'icona del lucchetto: 
![Immagine 2021-08-18 180018](https://user-images.githubusercontent.com/72148374/129931877-37ed3391-9813-4347-995b-b90954c44339.png)
a sinistra dell'indirizzo https, **eliminando l'eccezione** per l'accesso non sicuro, **il problema 401 sparisce!** 🎉

![Immagine 2021-08-18 094800](https://user-images.githubusercontent.com/72148374/129931449-a3215597-32b3-4fd6-8dfa-7b2e55990b36.png)

Lascio ugualmente i post nel caso possano tornare utili a qualcuno 😉

**Reply by snakuzzo** [reply]
Non serve, fidati. Cancello tutto perché sono temi affrontati mille volte quelli legati a certificati e https

**Reply by wilrino** [reply]
Dici? Io ho perso qualche ora a cercare info relative a questo problema e non lo avevo risolto (altrimenti non avrei scritto qui) 😁
Cmq ne sai sicuramente più di me.. probabilmente non ho perso abbastanza ore a cercare ;-)
